docs/edgeql/index.rst

```diff
@@ -4,8 +4,86 @@
 EdgeQL
 ======
 
-EdgeQL is the primary language of EdgeDB. It is used to define, mutate, and
-query data.
+EdgeQL is the primary language of EdgeDB. It is used to define, mutate,
+and query data. EdgeQL encompasses four items in general:
+
+**Querying language | SDL | DDL | Administration**
+
+Most frequently used in EdgeQL is the **querying language**. EdgeQL is
+a functional language in that every expression is a composition of
+one or more queries. The querying language looks like this:
+
+.. code-block:: edgeql
+
+    SELECT Issue {
+        number,
+        body,
+        due_date
+    }
+    FILTER
+        EXISTS .due_date
+    ORDER BY
+        .due_date
+    LIMIT 3;
+
+The simplicity of EdgeQL queries might remind you of GraphQL a bit,
+and EdgeQL does indeed have a similar straightforwardness while being
+more powerful
+(see some [comparisons of syntax here]() for details on that).
+
+
+**SDL (schema definition language)**. SDL is used to describe migrations
+to a specific schema state. In SDL you write the final types and
+relationships without needing to worry about making declarations in any
+particular order. Items in a schema written in SDL look like this:
+
+.. code-block:: sdl
+
+    type Movie {
+        required property title -> str;
+        required link director -> Person;
+        multi link actors -> Person;
+    }
+
+    type Person {
+        required property name -> str;
+    }
+
+- **DDL (data definition language)**. DDL is the less frequently used
+(but often convenient) cousin of SDL. DDL is used to transform schema
+step by step instead of all at once. The order is important in DDL
+because it involves issuing one command after another instread of
+describing the final form.
```

Comment:
```suggestion
**DDL (data definition language)**. DDL is the less frequently used
(but often convenient) cousin of SDL. DDL is used to transform schema
step by step instead of all at once. The order is important in DDL
because it involves issuing one command after another instread of
describing the final form.
```